german/the-black-hack.md

```diff
@@ -26,38 +26,39 @@ The Black Hack (TBH) ist ein traditionelles Tischrollenspiel, das mit Papier, St
 
 Wann immer Charaktere versuchen, etwas zu tun, das möglicherweise scheitern könnte, wird der Erfolg mit einer Probe auf einen Eigenschaftswert getestet. Um eine Probe erfolgreich zu bestehen, müssen Spieler mit einem W20 ***unter*** den entsprechenden Eigenschaftswert würfeln.
 
-Monster würfeln keine Proben - Spielercharaktere müssen ihren Angriffen ausweichen, indem sie eine Probe bestehen. Das einzige, was Monster würfeln, ist ihr Schaden.
+Monster würfeln keine Proben - Spielercharaktere müssen ihren Angriffen ausweichen, indem sie eine Probe bestehen. Das einzige, das Monster würfeln, ist ihr Schaden.
```

Comment:
Gefühlt geht beides. War bei Lars laut eigener Aussage auch eher Bauchgefühl.

Was mir dabei gerade so auf fällt: Müsste in den Kontext nicht "einzige" groß geschrieben werden?